sql/moz-fx-data-shared-prod/firefox_ios_derived/funnel_v1/query.sql

```diff
@@ -0,0 +1,102 @@
+WITH views_data AS (
+  SELECT
+    `date`,
+    territory AS country_name,
+    SUM(impressions_unique_device) AS views,
+  FROM
+    app_store.firefox_app_store_territory_source_type_report
+  WHERE
+    `date` >= '2022-01-01'
+  GROUP BY
+    `date`,
+    country_name
+),
+downloads_data AS (
+  SELECT
+    `date`,
+    territory AS country_name,
+    SUM(total_downloads) AS downloads,
```

Comment:
This should be `first_time_downloads`, right? Isn't that what Vanessa wanted?

Reply:
@bani I believe I kept the original naming from your query here. Would you prefer me to rename the field to `first_time_downloads` as suggested by Frank?

Reply:
no, I think we should keep total downloads. That's what works best with the previous and next funnel steps. Separating first from redownloads doesn't seem to be high priority with the stakeholders, since we don't have visibility on that down the funnel.

Reply:
ok, in this case I shall keep this as is. Thanks @bani for clarifying 🙇

Reply:
For the Android funnel, we're separating new and returning devices. It might be a good idea for us to do the same here: https://docs.google.com/presentation/d/14PAh2I-6hzweNHZH8QWDoUsq6FbnxowJBkGM0hlrh-s/edit#slide=id.p

Reply:
Made the change, thanks!